src/main/resources/site/processors/cms-plugin.js

```diff
@@ -1,26 +1,17 @@
-var contentLib = require('/lib/xp/content');
 var portalLib = require('/lib/xp/portal');
 
 exports.responseProcessor = function (req, res) {
-
     if (req.mode !== 'live') {
         return res;
     }
 
-    var content = portalLib.getContent();
-    var contentId = req.params.contentId;
-    if (!contentId) {
-        contentId = content._id;
+    const siteConfig = portalLib.getSiteConfig();
```

Comment:
I am not sure, it would work the same way as before.
In order for `portalLib.getSiteConfig()` to work application name must be set in request.

Reply:
It works fine, tested in GA, Metafields and this app